requirements.txt

```diff
@@ -3,10 +3,12 @@ asyncpg==0.27.0
 fastapi==0.95.2
 httpx==0.24.1
 jinja2==3.1.2
+jpype1==1.5.0
 pydantic==1.10.2
 pydantic-xml[lxml]==0.2.2
 python-dotenv==0.21.0
 setuptools-scm==7.0.5
 tortoise-orm==0.19.3
+tqdm==4.66.2
```

Comment:
Vu qu'on a besoin de jpype et tqdm uniquement pour le générateur gml, je ne suis pas sûr qu'il soit nécessaire de les rajouter comme pré-requis ici.

Reply:
Fixed in 7d34164
Don't mind the `break` in gml_generator, just a typo